### PR TITLE
[2.9] adding full k8s tests to airgap

### PR DESCRIPTION
**Pull request by slickwarren**
port of https://github.com/rancher/rancher/pull/47414

### CODE REVIEW

**Review by floatingman — APPROVED** [review]
LGTM

**Review by lscalabrini01 — DISMISSED** [review]

**Review by floatingman — APPROVED** [review]
LGTM

**Review by markusewalker — DISMISSED** [review]

File: pkg/controllers/provisioningv2/fleetcluster/fleetcluster.go
Comment (anupama2501) [inline pkg/controllers/provisioningv2/fleetcluster/fleetcluster.go:1]: fleetcluster files seem to be additional changes here compared to 2.8. is this expected?

**Review by floatingman — APPROVED** [review]
LGTM

**Pull request merged**